RedHatInsights/curiosity-frontend
# fix(platformServices): sw-625 restore onNavigate

## What's included

- fix(platformServices): sw-625 restore onNavigate

### Notes
- patch for routing, restores parts of the previous routing callback `insights.chrome.on('APP_NAVIGATION', callback)`, currently scheduled for 8:00 AM EST
- steps for release and testing
   - merge thie PS to stage-stable to minimally confirm working and no errors
      - if errors, start over
      - if no errors, continue
   - subsequent PR and merge to bypass checks and move to prod
      - if errors, rollback to GUI v2.9.0
      - if no errors, continue
   - rollback and remove the bypass commit
   - release to prod-beta as v2.10.1-rc.0
   - release to prod-stable with release commit and v2.10.1
      - rebase accordingly

- @mirekdlugosz @Hyperkid123 



## How to test






### In environment check, stage and prod
1. navigate and log into the appropriate environment
1. confirm the behavior of navigating between
   - RHEL and Satellite updates the product view accordingly
   - OpenShift Container and OpenShift Dedicated updates the product view accordingly

## Example

...

## Updates issue/story

sw-625
